sql/plan/load_data.go

```diff
@@ -144,9 +144,9 @@ func (l *LoadData) WithChildren(children ...sql.Node) (sql.Node, error) {
 	if len(children) != 1 {
 		return nil, sql.ErrInvalidChildrenNumber.New(l, len(children), 1)
 	}
-
-	l.Destination = children[0]
-	return l, nil
+	ret := *l
```

Comment:
You an avoid this rigmarole by declaring the receiver to a non-pointer type

Reply:
the bugs i run into are still usually because we were using a value instead of pointer type, if we overcorrect at some point we can always mass codegen all nodes as value types